samples/base/Core.hpp

```diff
@@ -76,6 +77,8 @@ namespace darwin {
         std::size_t _nbThread;
         std::size_t _cacheSize;
         std::size_t _threshold;
+        boost::atomic<FilterStatusEnum> _filter_status;
```

Comment:
Why not using std::atomic ?

Creating less dependencies to libs might be a good idea.

Reply:
done